app/services/migration/migrators/user.rb

```diff
@@ -33,10 +33,14 @@ def call
         trns = unique_validated_trns(ecf_user)
         validate_multiple_trns!(trns, user)
 
+        unique_emails = ecf_user.npq_applications.map { |app| app.participant_identity.email.to_s.downcase.strip }.compact.uniq
```

Comment:
Worth extracting this to a separate method to be consistent with `unique_validated_trns`?